apps/src/aichat/views/AichatView.tsx

```diff
@@ -179,6 +192,13 @@ const AichatView: React.FunctionComponent = () => {
 
   const onClear = useCallback(() => {
     dispatch(clearChatMessages());
+    dispatch(
+      addChatEvent({
+        timestamp: Date.now(),
+        descriptionKey: 'CLEAR_CHAT',
+        hideForParticipants: true,
+      } as ChatEvent)
```

Comment:
Same here as well

Reply:
Will remove here. No cast needed. I think I got in the habit of casting from cases above 😅